redwood-treehouse-guest/src/commonMain/kotlin/app/cash/redwood/treehouse/treehouseCompose.kt

```diff
@@ -55,6 +59,11 @@ private class RedwoodZiplineTreehouseUi(
    */
   override val scope = (treehouseUi as? ZiplineScoped)?.scope ?: ZiplineScope()
 
+  private val coroutineScope = CoroutineScope(
+    appLifecycle.coroutineScope.coroutineContext +
+      SupervisorJob(appLifecycle.coroutineScope.coroutineContext.job),
```

Comment:
Why this change? Doesn't this mean crashes in the composition won't correctly propagate?

Reply:
Yeah sorry the "this" was meant solely to be about the `SupervisorJob`. The subscope makes sense 👍